Framework/Crystal/src/PeakStatisticsTools.cpp

```diff
@@ -132,13 +132,15 @@ UniqueReflectionCollection::UniqueReflectionCollection(const UnitCell &cell, con
 /// the reflection family can not be found are ignored.
 void UniqueReflectionCollection::addObservations(const std::vector<Peak> &peaks) {
   for (auto const &peak : peaks) {
-    V3D hkl = peak.getHKL();
-    hkl.round();
+    V3D hkl = peak.getIntHKL();
+    V3D mnp = peak.getIntMNP();
 
-    auto reflection = m_reflections.find(m_pointgroup->getReflectionFamily(hkl));
+    if (mnp.norm2() == 0.0) {
```

Comment:
Looks like the statistics calculation ignores satellite peaks - I think it makes sense as the modulated lattice could have a different point group

Reply:
That's the intent. I don't think at this moment the symmetry algorithms can handle anything other than non-modulated structures.